### PR TITLE
switching intial ingest to priority queue

### PR DESCRIPTION
**Pull request by lcouzens**
Fixes [COST-2071](https://issues.redhat.com/browse/COST-2071)

Changes proposed in this PR:
* Switching to priority queue for initial ingest/summary to help new on boarding customers get there initial data sooner. 
* Really adding this for our QE tests so those don't randomly fail during a month when queue lengths are extremely long

Testing Instructions:
1. Add new source
2. See data process via priority queue
3. Also ensure day to day flow continues to not use priority queues.

NOTE:
Easiest way to test this is just stick `LOG.info(f"\n\n QUEUES: {SUMMARY_QUEUE}, {REPORT_QUEUE} \n\n")` after line 142, then just add a source see it log priority, then hit download again and see those change back to the usual queue names.

### CODE REVIEW

File: koku/masu/processor/orchestrator.py [hunk 1]
```diff
@@ -132,6 +133,13 @@ def start_manifest_processing(
                 files       - ([{"key": full_file_path "local_file": "local file name"}]): List of report files.
             (Boolean) - Whether we are processing this manifest
         """
+        with ProviderDBAccessor(provider_uuid) as accessor:
+            if accessor.get_setup_complete():
```
[the remark this answers is not in the document]
Comment (lcouzens) [reply koku/masu/processor/orchestrator.py:137]: You make a valid point, I think the resources _should_ be fine. That said I'm now wondering if we should only use the priority queue if the normal queues are not sub a certain count?

File: koku/api/provider/models.py [hunk 2]
```diff
@@ -181,7 +181,7 @@ def save(self, *args, **kwargs):
 
             LOG.info(f"Starting data ingest task for Provider {self.uuid}")
             # Start check_report_updates task after Provider has been committed.
-            transaction.on_commit(lambda: check_report_updates.delay(provider_uuid=self.uuid))
+            check_report_updates.s(provider_uuid=self.uuid, queue_name=PRIORITY_QUEUE).set(queue=PRIORITY_QUEUE)
```
Comment (maskarb) [inline koku/api/provider/models.py:184]: We still need this transaction, else this won't work correctly.
```suggestion
            transaction.on_commit(lambda: check_report_updates.s(provider_uuid=self.uuid, queue_name="priority").set(queue="priority")
```

I think the circular import is due do us importing this queue name.

**Comment by lcouzens** [comment]
smoke-tests passed and they used the priority queue when setting the schema!

**Pull request merged**